### ISSUE TITLE
Misleading text after activating console

### ISSUE THREAD

**Issue by eikowagenknecht** [openhab/openhabian#1312]
Using the current version, after enabling the remote console and setting the password a message box ist shown that ends with:

"The default password is habopen."

While technically true, the user just set a custom password one step before, so this is quite confusing and suggests that maybe the custom password was not used but the default one (which is not the case).

So I'd say just remove that last line :-)

**Comment by mstormi** [comment]
"default" means default. That's strange thinking I disagree with

**Comment by eikowagenknecht** [comment]
I don't think that this is strange thinking at all. While it may be perfectly clear to you as someone who knows the details of how the configuration tool works in and out, from a normal end users perspective the process is as follows:

1) Choose menu option to activate console
2) Be prompted to set a password (not optional!)
3) Be shown that the password's default was "habopen"

Why would anyone need that information at this time? There is no possibility that the password is still "hapoben" at the point in time where this message is shown. So it does not provide any benefit and for exactly that reason is confusing.

I can't speak for others of course but at least I thought "Well, I just set the password. But now it's showing me that the default was habopen. Does that mean that setting the password did not succeed and it used the default anyways?" I literally tried to login to the console using this default password to rule out the chance that it might have been used instead of my chosen password.

**Comment by mstormi** [comment]
Users can just leave the input blank. I'm sure most appreciate the default being mentioned, no matter if applicable.
So no I'm not gonna change it.

**Comment by eikowagenknecht** [comment]
Ok, so I understand it is possible to leave the field blank and then the default password is used (and not blank password)?

Then there are two scenarios:
1. User sets a non-default password. This is the scenario I described before and probably the common one. For this user it is completely irrelevant what the default password has been before and I don't think he will appreciate seeing this afterwards. Why would he?
2. User leaves the password field empty. For this user the default password indeed is a useful information. But then it should be shown one dialog earlier like: "If you leave the password field empty, the default password habopen will be set." That would also make it clear that it apparently is possible to set the default password by leaving the field empty. From the current text ("Please provide a secure password...") this is not to be known at all.

I'm not writing this to annoy you, but because I sincerely think the user experience can be improved here.

**Comment by mstormi** [comment]
> I'm not writing this to annoy you, but because I sincerely think the user experience can be improved here.

Or because you yourself misunderstood it so it must have been the instructions, not the user's understanding ?
As I said I disagree, but frankly I don't mind so I changed the wording.

**Comment by eikowagenknecht** [comment]
I understand that you obviously don't agree with the point I'm making here and that is fine. And if you think that this is too minor a detail to waste your time on, then just say so.

But I do **not** enjoy these aggressive comments from you and think they are quite inappropriate. It gives me the impression that you do not care about feedback but only want to prove that you are right. Which is an attitude that does not encourage people in helping you and does not suit an open source project like openHAB.

**Comment by mstormi** [comment]
> It gives me the impression that you do not care about feedback but only want to prove that you are right

I've spent more time replying than I think was adequate for this, showing I do care. And I don't have to prove anything.
You're the first in some years this text (btw it's not even authored by me) already exists that sees a need to change this.
Attitude, well. Yes you yourself came across pretty annoying and aggressive, too when I disagreed in the first place.
By experience, there's 2 kinds of people to respond like that: those who are truely altruistic, empathic and convinced to help others by their engagement (who then are somewhat disappointed when I don't support them in doing so) - or people that just want to be right _themselves_.
My "aggressive" comment was _in response to that_ and just a question that you can simply dismiss by answering "no".
So my apologies if you belong to the first group (which I really think you do).

**Comment by eikowagenknecht** [comment]
> You're the first in some years this text (btw it's not even authored by me) already exists that sees a need to change this.

I understand the point you're making, but I think two things factor in here: A) I'm very good at spotting inconsistencies and attuned to looking at UI details (it's my day job) and B) Most people will - if they notice it - not be enganged enough to actually work on changing it (by that I mean opening an issue here for example). If you look at my Github profile you will see that I opened various openHAB and other open source project related issues and quite some of them address problems that were years old but still not fixed or officially documented. I've stopped relying on "someone else will take action on that by themselves" quite some time ago.

Regarding the rest of your comment: I accept the apology and I'm sorry if I came across as aggressive to you. That was never my intention. I really appreciate the work you and the other maintainers do and opening Github issues when I see that things can be improved is my way of helping (apart from the Foundation membership ;-)). So again: I don't do this to annoy you but to make openHAB/ian a better experience for everyone.

Now let's stop spending time on this discussion and continue making openHAB the greatest smart home platform ;-)
